src/hydro_system.hpp

```diff
@@ -1036,6 +1040,8 @@ void HydroSystem<problem_t>::ComputeFluxes(amrex::MultiFab &x1Flux_mf, amrex::Mu
 		sR.cs = cs_R;
 		sR.E = E_R;
 		sR.Eint = Eint_R;
+		sR.by = 0.0;
+		sR.bz = 0.0;
```

Comment:
Same comment as above -- can you add a comment in the source here that explains that you've just set the magnetic fields to zero for testing purposes?